docs/user/library/data/arcsde.rst

```diff
@@ -1,7 +1,9 @@
 ArcSDE Plugin
 -------------
 
-The ArcSDE plugin allows the GeoTools developer to work with the `ArcSDE <http://en.wikipedia.org/wiki/ArcSDE>`_ middle ware. The ArcSDE plugin has been with us since GeoTools 2.0 and is much loved.
+Unsupported plugin allowing to work with the `ArcSDE <http://en.wikipedia.org/wiki/ArcSDE>`_ middle ware.
+The ArcSDE plugin has been with us since GeoTools 2.0 and is much loved, yet it has become unsupported
```

Comment:
A plugin with sentiment! (that must be a first)

Reply:
lol, didn't want to change the existing statement :)